Jenkinsfile_CNP

```diff
@@ -74,8 +74,13 @@ def vaultOverrides = [
 // Assume a feature build branched off 'develop', with dependencies develop-to-develop.
 env.DEFINITION_STORE_URL_BASE = "https://ccd-definition-store-api-${definitionStoreDevelopPr}.service.core-compute-preview.internal".toLowerCase()
 env.CCD_DATA_STORE_API_BASE_URL = "https://ccd-data-store-api-${dataStoreApiDevelopPr}.service.core-compute-preview.internal".toLowerCase()
+
 // NB: this is the callback URL used when testing NOC so its an internal call: data-store.preview -> mca.preview.
-env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-${env.BRANCH_NAME}-java"
+if (env.BRANCH_NAME == 'master' || env.BRANCH_NAME == 'demo' || env.BRANCH_NAME == 'perftest' || env.BRANCH_NAME == 'ithc' || prsToUseAat.toLowerCase().contains(env.BRANCH_NAME.toLowerCase())) {
+  env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-aat.service.core-compute-aat.internal"
+} else {
+  env.MCA_API_BASE_URL = "http://aac-manage-case-assignment-${env.BRANCH_NAME}-java.aac"
```

Comment:
I think this can only be safely set to the PR/BRANCH_NAME version of the MCA APIs when the Data Store API is running in isolation.  Otherwise the tests risk being affected by parallel pipelines running in against any CCD Repo that re-publishes the common Test Defintions.

Reply:
👍 CALLBACK_FTA_ENABLED process should now resolve this